src/App.tsx

```diff
@@ -1,25 +1,74 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import Header from "./components/Header";
+import Game from "./components/Game";
+import StartGame from "./components/StartGame";
+import { useState } from "react";
+import DisplayHighscore from "./components/DisplayHighscore";
+
+type GameProps = {
+  id: string;
+  length: number;
+};
+export type HighscoreObject = {
+  id: string;
+  multi: boolean;
+  length: number;
+  correctWord: string;
+  guesses: string[];
+  startTime: string;
+  endTime: string;
+  name: string;
+};
```

Comment:
This type is used in lots of places, so I would put it somewhere more "general" than inside this particular component file. Of course the `App` component is not just any component, but it's still not normally a thing that you import all over the place, whereas this type is.